README.md

```diff
@@ -36,9 +36,17 @@ Just create a project, and you’re good to go.
 
 Install it once globally:
 
-```sh
-npm install -g create-react-app
```

Comment:
IMO that's not as good because people may blindly copy everything in a multiline code block and paste it into their terminal.